calibr8/contrib/normal.py

```diff
@@ -307,11 +317,19 @@ def predict_dependent(self, x, *, theta=None):
         """
         if theta is None:
             theta = self.theta_fitted
-        mu = core.exponential(x, theta[:2])
+
+        if self.fixed_intercept is None:
+            theta_mu = (theta[0], theta[1], theta[2])
```

Comment:
Suggestion: theta[:3]

Reply:
I did this because the direct indexing should lead to fewer operations in the symbolic graphs.
The `calibr8.exponential` unpacks the `theta` so it's either:
* Tensor-slicing into a Vector, then Tensor-subindexing (unpacking)
or
* Tensor-subindexing, but unpacking the tuple is pure python (the code above)